pkg/translate/translate_value_test.go

```diff
@@ -365,8 +365,9 @@ func TestNewReverseTranslator(t *testing.T) {
 		{
 			name:         "version 1.5",
 			minorVersion: version.NewMinorVersion(1, 5),
-			wantVer:      "1.4",
-			wantErr:      false,
+			//TODO Once we add translate configs for 1.5, update wantVer to "1.5"
```

Comment:
Could you please create an issue and assign to @richardwxn ?

Reply:
sure, issue created: https://github.com/istio/istio/issues/19801

Reply:
@ostromart